bitmap-team0/scratch.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+
+const fs = require('fs');
+const bitmap = fs.readFileSync(`${__dirname}/../img/palette-bitmap.bmp`);
+const bmp = {};
+
+bmp.type = bitmap.toString('utf-8', 0, 2);//BM
+bmp.size = bitmap.readUInt32LE(2);//11078
+bmp.reserved_one = bitmap.readUInt32LE(6);//0
+bmp.reserved_two = bitmap.readUInt32LE(8);//70647808 ?
+bmp.pixel_array_start = bitmap.readUInt32LE(10);//1078
+bmp.header_size = bitmap.readUInt32LE(14);//40 BITMAPINFOHEADER
+bmp.width = bitmap.readUInt32LE(18);//100
+bmp.height = bitmap.readUInt32LE(22);//100
+bmp.color_planes = bitmap.readUInt32LE(26);//524289
+bmp.bpp = bitmap.readUInt32LE(28);//bits per pixel
+bmp.compression_method = bitmap.readUInt32LE(30);// 0 none
+bmp.image_size =  bitmap.readUInt32LE(34);//10000
+bmp.horizontal_resolution = bitmap.readUInt32LE(38);//2834
+bmp.vertical_resolution = bitmap.readUInt32LE(42);//2834
+bmp.number_of_colors = bitmap.readUInt32LE(46);//256
+bmp.important_colors = bitmap.readUInt32LE(50);//256
```

Comment:
@bazacko @bhavyaab @ShaAllFar these should be created as a constructor instance and not directly on an object literal like the one defined above - that will allow for more modularity and reusability